components/highlight.js

```diff
@@ -57,6 +59,28 @@ const Highlight = ({ research }) => {
           </Box>
         </Column>
       </Row>
+      <Row as='figure' sx={{ mt: [4, 5, 6, 7], mb: [4, 4, 4, 5] }}>
+        <Column start={[1, 1, 2, 3]} width={[6, 8, 10, 8]}>
+          <Link href={research.href}>
+            <Image
```

Comment:
Perhaps depending on how complicated the height-setting logic becomes (e.g. maybe less necessary if there's a simple solution using `aspect-ratio`), we might consider formalizing this `Image` pattern by building into `components`.

Reply:
yeah maybe we wait for at least a third use case? there's a lot here that's pretty specific to these images (and their aspect ratios) and these layouts (although the two cases have a lot in common, not clear yet whether that will generalize)